Add Charles Ejiegbu

keleris32 opened the pull request:
`[Add profile] -> [Charles Ejiegbu]`

## Add Charles Ejiegbu
  

<!--🙏🏾 =============================== CUSTOM PR ========================================== 🙏🏾-->
### What does this PR do?:
Add my profile to the contributor's list

Mannuel25 reviewed (changes requested):
Hello Charles, thanks for contributing. Some changes have been made since your last edit, could you pull the changes made to avoid conflicts, then add your own profile.
If you've any issues doing that, kindly let me know. :wink:

keleris32 commented:
> Hello @keleris32 , thanks for contributing. Some changes have been made since your last edit, could you pull the changes made to avoid conflicts, then add your own profile. If you've any issues doing that, kindly let me know. 😉

Thank you for the feedback. I have fixed the merge conflicts and updated the comment to add my own profile. Please let me know if there are further adjustments to make. Thank you

Mannuel25 reviewed (approved):
Awesome job @keleris32 !
